garrysmod/html/js/menu/control.Servers.js

```diff
@@ -262,7 +262,9 @@ function AddServer( type, id, ping, name, desc, map, players, maxplayers, botpla
 
 	data.hasmap = DoWeHaveMap( data.map );
 
-	data.recommended = data.ping;
+	data.recommended = 40;

```

Comment:
I don't think 40 is necessarily ideal. I'd say more like 20.

Reply:
data.recommended default value 20 vs 40 will have no difference. As long as the ping is under 60 all the servers will have data.recommended pre-playercount etc. calculation set to 40

Whether the number being used is 40 or 20 does not matter.